LocalSettings.php

```diff
@@ -117,13 +117,35 @@
 		'default' => false,
 	],
 	'wgPivotFeatures' => [
```

Comment:
You'll need to set this in LocalWiki.php using $wi->config->settings['wgPivotFeatures']['default']

this is because you are using variables that won't be set in time.

Reply:
you can always do +<dbname> so it latches onto the default + overrides. (this is for where you do thegreatwarwiki)

Reply:
Done

Reply:
Copy from other conversation https://github.com/Universal-Omega/mw-config/blob/0427a0b205161670043f46df1202b2098f03c54e/LocalSettings.php#L1833 does work with variables.